test/js/third_party/esbuild/esbuild-test.js

```diff
@@ -82,5 +82,3 @@ import { build, buildSync, transform, transformSync } from "esbuild";
     throw new Error("Test failed.");
   }
 }
-
-process.exit(0);
```

Comment:
probably was not needed, but a reason to keep this is that the process should exit since there are no ref'd polls.